moneybird.js

```diff
@@ -1,21 +1,124 @@
 const SuperAgent = require('superagent');
 const {MONEYBIRD_API_TOKEN, MONEYBIRD_ADMINISTRATION_ID} = process.env;
+const {keyBy} = require('lodash');
+const {ENV} = process.env;
+const Fetch = require('./fetch.js');
+const DevFetch = require('./devFetch.js');
+const yaml = require('js-yaml');
 
-exports.getContactData = async (contactId) => {
+const use_cache = false;
 
-    const contact_request = await SuperAgent
-        .get(`https://moneybird.com/api/v2/${MONEYBIRD_ADMINISTRATION_ID}/contacts/${contactId}`)
-        .set('Authorization', `Bearer ${MONEYBIRD_API_TOKEN}`)
-        .set('Accept', 'application/json')
-    return contact_request.body
+var fetch;
+switch (ENV) {
+    case 'prod':
+    case 'test':
+        fetch = new Fetch();
+        break;
+    case 'dev':
+    default:
+        fetch = new DevFetch();
```

Comment:
What does this do exactly?

Reply:
I just copied this from `server.js`, since I started with working there, but after a while I decided to split code off to `moneybird.js`.

Reply:
Please see `fetch.js` and `devFetch.js` for more context